vaadin-grid-flow-demo/src/main/java/com/vaadin/flow/component/treegrid/demo/TreeGridDemo.java

```diff
@@ -121,8 +125,7 @@ private void createLazyLoadingTreeGridUsage() {
         grid.addHierarchyColumn(Account::toString).setHeader("Account Title");
         grid.addColumn(Account::getCode).setHeader("Code");
 
-        HierarchicalDataProvider dataProvider =
-                new AbstractBackEndHierarchicalDataProvider<Account, Void>() {
+        HierarchicalDataProvider dataProvider = new AbstractBackEndHierarchicalDataProvider<Account, Void>() {
```

Comment:
I don't think this is necessary. Please keep the indentation.

Reply:
Done